cohttp-lwt-unix/src/client.ml

```diff
@@ -1,3 +1,12 @@
-include Cohttp_lwt.Make_client (Io) (Net)
+module Sleep = struct
+  let sleep_ns ns =
+    Lwt_unix.sleep @@
+    let open Int64 in
+    if ns >= zero
+    then to_float ns
```

Comment:
Here you need to divide by a million, no?

Reply:
oh, indeed. need do divide by 10⁹.